### PR TITLE
[com_templates] - support for scss,sass source formats on joomla update

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #10186 .

### Summary of Changes
 added support support for scss,sass source formats on update

see #12712 for more info

### CODE REVIEW

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -1933,4 +1934,70 @@ private function cleanJoomlaCache()
...
+		}
+		catch (JDatabaseExceptionExecuting $e)
+		{
+			echo JText::sprintf('JLIB_DATABASE_ERROR_FUNCTION_FAILED', $e->getCode(), $e->getMessage()) . '<br />';
+
+			return;
+		}
+
+		$params         = json_decode($params, true);
+		$sourceformats  = explode(',', $params['source_formats']);
```
Comment (andrepereiradasilva) [inline administrator/components/com_admin/script.php]: you probably could replace all above code by just:
```php
$sourceformats = explode(',', JComponentHelper::getParams('com_templates')->get('source_formats', 'txt,less,ini,xml,js,php,css,scss,sass'));
```

**Comment by wilsonge** [comment]
I actually think this is wrong. If people want to retrospectively add this they can. But imho we shouldn't be changing this for them (ditto for the media manager params)

**Comment by mbabker** [comment]
The other thing is looking at this, if someone removed these items from their configuration, an update would restore them.  So there probably isn't a good way to do this even if we wanted to.

**Comment by alikon** [comment]
closing as is too late for doing it now
thanks all for your feedback

**Pull request closed without merging**